tests/synthdnn/README.md

```diff
@@ -0,0 +1,28 @@
+# Synthdnn
+
+Synthdnn is a suite of scripts for collecting and analyzing oneDNN performance
+across a randomly generated data. The general architecture is intended to follow
+a data pipeline composed of synthetic problem generation, data collection, and
+data analysis. The `synthdnn.py` script provides a command line interface to
+these tools. Sample Usage:
+
+
+Data Generation:
+```
+python3 synthdnn.py <primitive> [sampling controls] -b <batch_file>
+```
```

Comment:
Please specify a fixed line that is used to generate `test_matmul_random_ci`.

I also think that `test` is over-used across the validation. Let's make it `harness_matmul_generated_ci` and include that into `test_matmul_ci`.

Reply:
I modified the script to log the generation command in the generated file. I don't intend to include the command here as that seems less flexible and maintainable  (for example, if we start using multiple generated test suites).